<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-03T16:19:31 SAST" name="Suite" finished-at="2022-03-03T16:21:35 SAST" duration-ms="123943">
    <groups>
    </groups>
    <test started-at="2022-03-03T16:19:31 SAST" name="Test" finished-at="2022-03-03T16:21:35 SAST" duration-ms="123943">
      <class name="org.bongz.tests.MultiplyTests">
        <test-method is-config="true" signature="setUp([Ljava.lang.Object;)[pri:0, instance:org.bongz.tests.MultiplyTests@383bfa16]" started-at="2022-03-03T16:19:32 SAST" name="setUp" finished-at="2022-03-03T16:20:28 SAST" duration-ms="55757" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.Object;@5ee34b1b]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginlogoutTest(java.util.Map)[pri:0, instance:org.bongz.tests.MultiplyTests@383bfa16]" started-at="2022-03-03T16:20:28 SAST" name="loginlogoutTest" description="To check if the user can successfully login and logout" data-provider="getData" finished-at="2022-03-03T16:21:34 SAST" duration-ms="65745" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=QEtCVG9remFuMjAyMQ==, menutxt=, browser=chrome, testname=loginlogoutTest, execute=yes, version=98.0, username=bomaseko11}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expecting:
 <"Login">
to be equal to:
 <"My Points">
but was not.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expecting:
 <"Login">
to be equal to:
 <"My Points">
but was not.
at org.bongz.tests.MultiplyTests.loginlogoutTest(MultiplyTests.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginlogoutTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:org.bongz.tests.MultiplyTests@383bfa16]" started-at="2022-03-03T16:21:34 SAST" name="tearDown" finished-at="2022-03-03T16:21:35 SAST" duration-ms="1219" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.bongz.tests.MultiplyTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
